### ISSUE TITLE
Update ValidateScript ErrorMessage in reference and conceptual documentation

### ISSUE THREAD

**Issue by peetrike** [MicrosoftDocs/PowerShell-Docs#9252]
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

- https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/validatescript-attribute-declaration?view=powershell-5.1
- https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/developer/cmdlet/validatescript-attribute-declaration.md

### Summary

The page states, that `ValidateScript` attribute has optional **ErrorMessage** parameter.  But in PowerShell 5.1 it does not have that parameter.  That parameter was added in PowerShell 6.x

### Details

in PowerShell 5.1:
```powershell
function simpletest {
    param (
        [ValidateScript({ $_ -gt 3 }, ErrorMessage = 'must be bigger than 3')] $Value
    )
    'The value was: {0}' -f $Value
}
simpletest -Value 4
```
```output
Property 'ErrorMessage' cannot be found for type 'System.Management.Automation.ValidateScriptAttribute'.
At line:3 char:9
```

### Suggested Fix

Remove redundant parameter description.

**Comment by peetrike** [comment]
Or if there is no version-specific pages, the parameter description should have statement about version requirements.

**Comment by michaeltlombardi** [comment]
@peetrike - good catch, thank you! Only the reference content is per-version. I'll get a note added to this document.

I checked the reference documentation just to be safe, and in [about_Functions_Advanced_Parameters][5.1] for Windows PowerShell, there's no reference to the **ErrorMessage** parameter, which is correct. However, it's also not included in the later versions ([7], [7.2], [7.3]) so I'll correct those as well.

[5.1]: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-5.1#validatescript-validation-attribute
[7]: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7#validatescript-validation-attribute
[7.2]: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.2#validatescript-validation-attribute
[7.3]: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.3#validatescript-validation-attribute
